### ISSUE TITLE
Wails Exit Status 1

### ISSUE THREAD

**Issue by matt9mg** [wailsapp/wails#2378]
### Description

wails dev no longer works and seems to present an empty error with status 1

### To Reproduce

mt@macbook rawnet-desktop-app % wails dev -v=2
Wails CLI v2.3.1

Executing: go mod tidy
  • Generating bindings:
  ERROR

          exit status 1
 ♥   If Wails is useful to you or your company, please consider sponsoring the project:
https://github.com/sponsors/leaanthony

mt@macbook rawnet-desktop-app % node --version
v16.19.0

mt@macbook rawnet-desktop-app % npm --version
8.19.3

mt@macbook rawnet-desktop-app % go version
go version go1.19.4 darwin/amd64

### Expected behaviour

wails dev compiles

### System Details

```shell
mt@macbook rawnet-desktop-app % wails doctor
Wails CLI v2.3.1

Scanning system - Please wait (this may take a long time)...Done.

# System

OS           | MacOS
Version      | 12.6
ID           | 21G115
Go Version   | go1.19.4
Platform     | darwin
Architecture | amd64

# Wails

Version | v2.3.1

# Dependencies

Dependency                | Package Name | Status    | Version
Xcode command line tools  | N/A          | Installed | 2395
npm                       | N/A          | Installed | 8.19.3
*Xcode                    | N/A          | Available |
*upx                      | N/A          | Available |
*nsis                     | N/A          | Available |
* - Optional Dependency

# Diagnosis

Your system is ready for Wails development!
Optional package(s) installation details:
  - Xcode: Available at https://apps.apple.com/us/app/xcode/id497799835
  - upx : Available at https://upx.github.io/
  - nsis : More info at https://wails.io/docs/guides/windows-installer/

 ♥   If Wails is useful to you or your company, please consider sponsoring the project:
```

**Comment by stffabi** [comment]
Thanks for using Wails and opening this issue.

During binding generation a special build of your app is created and run. It seems like this special build is triggering an error.
We use the build tag `bindings` for this purpose, does your app run without an error when building with  `go build -tags bindings` and it is executed?

**Comment by matt9mg** [comment]
Hi @stffabi 

It's a great project :)

mt@macbook rawnet-desktop-app % go build -tags bindings

no error but it I try run

go run changme = exit status 1

if i try run that as an executable nothing happens (i.e. chmod +x changme && ./changeme)

Also running wails dev again I get the same exit code.

Sorry if my debug report is incorrect or if I've missunderstood your comment, if it helps my colleague can run the app on his machine with the same debug settings I posted in my original report, so it makes me think its something machine related but I have no idea what as myself and a colleague who can run it have matched wails doctor repot.

**Comment by stffabi** [comment]
Thanks for this information.

Maybe you could try to debug your application with the debugger when using  `-tags bindings` to see where the exit code comes from.

**Comment by matt9mg** [comment]
hi @stffabi thank you for the advice.

Sorry if this is an od thing to say but would this be the command to test with?

`wails dev -tag bindings` to see where it dies?

**Comment by stffabi** [comment]
My idea was more to start the binary within your IDE for debugging, most likely with the delve debugger. I assume there are  guides how to use a custom tag for debugging in your IDE.

**Comment by stffabi** [comment]
@matt9mg just wanted to check back if you were able to solve this issue?

**Comment by Nitamet** [comment]
I've got the same problem now.

I was working on code for self-restarting after self-updates and accidentally wrote restart in the loop, which caused my `wails dev` running application to restart many times. Seems like that restarting somehow lead to the problem.

I removed that loop, but now, when generating bindings, I am getting `"exit status 1"` with no error description on any project - even on older versions that worked before and on clean templates. It indeed seems to be machine-related issue, because I can run wails dev on my second OS without any problem, even with the same project that broke my Wails CLI on the first OS.

I've reinstalled Wails CLI, but it didn't help.
My first OS is Linux, second OS is Windows.
